DESCRIPTION

```diff
@@ -49,6 +49,7 @@ Imports:
     magrittr,
     memoise,
     methods,
+    progress,
```

Comment:
I wonder if we should add a `progress = NA` argument to the functions that implement a progress bar. This would allow moving the package to "Suggests". Behavior:

- If `NA`, hide in non-interactive mode, show in interactive mode, give a message if package not installed
- If `FALSE`, hide
- If `TRUE`, fail if package is not installed

What do you think?